src/components/Prediction/index.js

```diff
@@ -4,6 +4,14 @@ import styled from "styled-components"
 
 const Match = styled.span`
   font-weight: bold;
+  width: 100%;
+  display: block;
```

Comment:
This should not be `display: block;` in my opinion since it forces the bold text to be on its own line. 

![image](https://user-images.githubusercontent.com/8575456/54036971-91d94500-4182-11e9-83e5-120d091d96b5.png)

Reply:
`Flex` will be a better option in  here i guess.

Reply:
`Flex` would work, my question is if we need `display: 'block'` at all with you setting width to 100%. I never really experienced the issue outlined in #37 #38 .

Could you see if it still works with removing this property?